alert/topics.go

```diff
@@ -11,19 +11,23 @@ import (
 )
 
 const (
-	// eventBufferSize is the number of events to buffer to each handler per topic.
-	eventBufferSize = 5000
+	// DefaultEventBufferSize is the default number of events to buffer to each handler per topic.
+	DefaultEventBufferSize = 5000
 )
 
 type Topics struct {
-	mu sync.RWMutex
-
-	topics map[string]*Topic
+	mu              sync.RWMutex
+	eventBufferSize int
+	topics          map[string]*Topic
 }
 
-func NewTopics() *Topics {
+func NewTopics(bufferSize int) *Topics {
```

Comment:
Should this be a method of the Topics struct so the buffer size is automagically available?

Reply:
I don't think that would be useful, as it is only ever instantiated once at startup and never changed.